frontend/rust-lib/flowy-server/src/af_cloud/impls/search.rs

```diff
@@ -32,7 +32,7 @@ where
     // Filter out irrelevant results
     let result = result
       .into_iter()
-      .filter(|r| r.score < SCORE_LIMIT)
+      .filter(|r| r.score > SCORE_LIMIT)
```

Comment:
Score limit needs adjustment. Right now it's 0.8, which will be very strict after switching the sings. I propose `0.3` as a new default.

Reply:
Understood